src/deploy-cromwell-on-azure/scripts/helm/templates/identity.yaml

```diff
@@ -4,3 +4,4 @@ metadata:
   annotations:
     azure.workload.identity/client-id: {{ .Values.identity.clientId }}
   name: {{ .Values.identity.name }}-sa
+  namespace: {{ .Values.config.coaNamespace }}
```

Comment:
I'm curious was the explicit namespace required now for some reason?

Reply:
It appears that some bugfix in the authentication library is requiring more stars to align before authentication is possible. This was one of the things I had to add before I had consistent successful results.